src/styles/blocks/header.scss

```diff
@@ -0,0 +1,65 @@
+.page__header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  margin: 0 auto;
```

Comment:
I know in this task we didn't look at the header but:
This doesn't do anything since it always stretches to whole page width. I think you should add a max-width so that it there is no huge gap between logo and other stuff in a wider screen. Say 1200px or whatever you want.